templates/master/00-master/on-prem/files/haproxy-haproxy.yaml

```diff
@@ -20,7 +20,7 @@ contents:
       bind :::{{`{{ .LBConfig.LbPort }}`}} v4v6
       default_backend masters
     listen health_check_http_url
-      bind :::30936 v4v6
+      bind :::20936 v4v6
```

Comment:
Shouldn't this be in the 9000-9999 range and ideally be specifically reserved (as node exporter and CVO are)?

Reply:
I had left this out of the range because I thought it could be changed to listen on only localhost, but when I tried that it doesn't work so yeah, I'll have to change it.